paddle/framework/channel_test.cc

```diff
@@ -78,3 +78,19 @@ TEST(Channel, ConcurrentSendNonConcurrentReceiveWithSufficientBufferSize) {
   t.join();
   delete ch;
 }
+
+TEST(Channel, SimpleUnbufferedChannelTest) {
+  auto ch = MakeChannel<int>(0);
+  std::thread t([&]() {
+    // Write one integer to channel
+    int send = 25;
+    ch->Send(&send);
+  });
+  int recv;
+  ch->Receive(&recv);
+  EXPECT_EQ(recv, 25U);
```

Comment:
To ensure that `UnbufferedChannel` works well, I think that we should call `Send` and `Receive` many times.
e.g.
```
  size_t sum_send = 0;
  std::thread t([&]() {
    for (size_t i = 1; i <= 3; ++i) {
      ch->Send(&i);
      sum_send += i;
    }
  });
  size_t sum_rece = 0;
  size_t recv;
  for (size_t i = i; i <= 3; ++i) {
    EXPECT_EQ(sum_send, sum_rece);
    ch-> Receive(&recv);
    EXPECT_EQ(recv, i);
    sum_rece += i;
    std::this_thread::sleep_for(std::chrono::milliseconds(100));
  }
```

Reply:
Thanks so much for the feedback! We will be definitely adding more test cases to this file like the one you mentioned here.

Reply:
@chengduoZH Thank you for the feedback. Fixed it!